aws_eventbridge/eventbridge.yaml

```diff
@@ -0,0 +1,203 @@
+AWSTemplateFormatVersion: "2010-09-09"
+Parameters:
+  ApiKey:
+    Description: >-
+      Your Datadog API Key
+    Type: String
+    AllowedPattern: .+
+    ConstraintDescription: ApiKey is required
+    NoEcho: true
```

Comment:
Are we planning on autofilling this template with a customer selected APIKey? Or linking to it from the app / docs without filling in the API key. If we want to fill in the api key automatically from the frontend I think we'll need to remove NoEcho (you can't pass parameter values through the url if they have NoEcho), but if we aren't doing that we can keep this.

Reply:
We won't be autofilling it for now.